src/trace/server_timing/metric.rs

```diff
@@ -80,6 +80,7 @@ mod test {
     use std::time::Duration;
 
     #[test]
+    #[allow(clippy::redundant_clone)]
```

Comment:
why allow instead of resolve? is this more complex than removing the last .clone()?

Reply:
Makes it more consistent to read.

Reply:
This one was probably catched by your editor but not by the CI which does not clippy the tests.
Maybe you could consider using `clippy --test` instead of `clippy` in the the CI to catch those issues earlier ?

Reply:
Huh? I run the same command as CI runs.

Reply:
Strange indeed, On my side I only get the redundant closure error when running `cargo clippy --tests`. Running rust stable 1.45.2 and nightly 1.47.0-nightly (f44c6e4e2 2020-08-24).

Reply:
Forgot to say I am running Windows 10, I will try on Linux tonight

Reply:
Investigating this further I see that `cargo clippy --all-targets` also activate the `--tests` flag. This is maybe the command you are running ? When running only `cargo clippy -- -D warnings` (used by CI) I do not get the errors from `#[cfg(test)]` guarded code.

Reply:
```
nightly-x86_64-pc-windows-msvc (overridden by +toolchain on the command line)
rustc 1.47.0-nightly (5180f3da5 2020-08-23)
```

```
cargo +nightly clippy --tests --examples -- -D warnings
```

Huh, turns out that's what Tide runs but not this. Adding a commit...